backend/core/scripts/minimal-listing.json

```diff
@@ -5,6 +5,7 @@
   "listingMultiselectQuestions": [],
   "buildingAddress": {
     "city": "Oakland",
+    "county": "Alameda",
```

Comment:
jw where do we use these scripts?

Reply:
I'm not sure of the history of this one either, but the Detroit batch import scripts I've been working on live in this directory, so I figured this is part of a script that has been used along the way at some point and figured it would need county if it's ever used again.

Reply:
I just looked this up, this is used as a sample json file for an import script.